src/tactic/ui/panel/hash_panel_wdg.py

```diff
@@ -162,11 +162,34 @@ def _get_predefined_url(cls, key, hash):
                 # put in a check to ensure this is a user
                 parts = link.split(".")
 
+                '''For usernames with at most one period
                 # you can only see your personal links
                 user = Environment.get_user_name()
                 if user == parts[0]:
                     personal = True
+                elif parts[1]:
```

Comment:
You are right, the solution for one period needs to check the index. 

The second solution (which is uncommented one) handles arbitrary number of indexes, no?